internal/messaging/service.go

```diff
@@ -34,6 +37,7 @@ var (
 	_ Service = (*partnerService)(nil)
 	_ Service = (*pingService)(nil)
 	_ Service = (*transportService)(nil)
+	_ Service = (*seatMapService)(nil)
 )
```

Comment:
Don't we need the other two services here?

Reply:
Added